### DISCUSSION TITLE
dcm4chee-arc is gathering the records of different patients

### DISCUSSION THREAD

**Discussion by medicaltecnologia** [dcm4che/dcm4chee-arc-light#3084] [Q&A] [answered]
same patient id. dcm4chee-arc is gathering the records of different patients. is there any configuration in dcm4che-arc that block or create separate exams? please, could someone help me solve the problem

**Comment by vrindanayak** [comment, marked as answer]
You may choose to either :
- [Optionally validate Patient Attributes of received instances against Patient Attributes in previous received instances of the Patient](https://github.com/dcm4che/dcm4chee-arc-light/issues/1990) by configuring [Reject Conflicting Patient Attribute(s)](https://dcm4chee-arc-cs.readthedocs.io/en/latest/networking/config/archiveNetworkAE.html#dcmrejectconflictingpatientattribute)
- [Optionally use all Patient attributes for identifying Patients without Issuer of Patient ID](https://github.com/dcm4che/dcm4chee-arc-light/issues/2851) by configuring [Identify Patient by all Attributes](https://dcm4chee-arc-cs.readthedocs.io/en/latest/networking/config/archiveDevice.html#dcmidentifypatientbyallattributes)
- [Optionally validate Patient ID of received instances against the Patient ID in previous received instances of the Study](https://github.com/dcm4che/dcm4chee-arc-light/issues/506) by configuring [Accept Conflicting Patient ID](https://dcm4chee-arc-cs.readthedocs.io/en/latest/networking/config/archiveNetworkAE.html#dcmacceptconflictingpatientid) value to `NO`

**Comment by medicaltecnologia** [comment]
sorry, i'm new to dcm4chee-arc.
please could you guide me the way where i can register this rule
dcmAcceptConflictingPatientID


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Livre
de vírus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>.
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Em seg., 15 de mar. de 2021 às 12:47, Vrinda ***@***.***>
escreveu:

**Reply by vrindanayak** [reply]
If you already have an archive running, 
- Go to `Configuration` page -> `Devices` tab -> Select `dcm4chee-arc` device
- Go upto its `Archive Device Extension` and search for `Accept Conflicting Patient ID` and change the value from default `MERGED` to `NO`

![image](https://user-images.githubusercontent.com/15785031/111188220-1d2d1280-85b5-11eb-88a7-cd38c7f8c4b8.png)

**Comment by medicaltecnologia** [comment]
I applied the Accept Patient in Conflict rule, but it doesn’t work the
patients have different names but the patient ids are the same.

Is there a possibility that dcm4chee-arc does not group studies with
the same patient id?


Em seg., 15 de mar. de 2021 às 13:37, Vrinda ***@***.***>
escreveu:

**Comment by medicaltecnologia** [comment]
thanks,
I already managed, follow the link below that I used to fix


https://github.com/dcm4che/dcm4chee-arc-light/issues/1991

Em seg., 15 de mar. de 2021 às 15:05, Anderson Sousa ***@***.***>
escreveu:
